cmd/context/utils.go

```diff
@@ -369,3 +369,21 @@ func fileExistsAndNotDir(filename string) bool {
 	}
 	return !info.IsDir()
 }
+
+func GetOktetoManifestPath(path string) string {
```

Comment:
we should also support to specify the path with `-f`

Reply:
and the `--namespace` flag

Reply:
The namespace flag, value by default according to the spec is the value specified at the manifest. we currently havent got implemented this field on the manifest, should i open a pr to add this field to manifest ? if not, i can implement the flag and default namespace be the one in the context?

Reply:
yes to both. the order is flag > manifest > context. in the manifest is also optional